src/praisonai/praisonai/framework_adapters/workflow_framework.py

```
@@ -0,0 +1,42 @@
+    if not framework or str(framework).lower() == "praisonai":
+        return
+
+    supported = ""
+    try:
+        from .registry import list_framework_choices
+
+        choices = [f for f in list_framework_choices(include_unavailable=True) if f != "praisonai"]
+        if choices:
+            supported = f" Available registered frameworks: {', '.join(sorted(choices))}."
+    except Exception:
+        pass
```

Ruff (0.15.18)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)